assets/sass/_hubspot.scss

```diff
@@ -91,7 +91,16 @@
 
     &.pricing {
         label {
-            @apply font-normal;
+            @apply font-normal text-base;
+        }
+        input, textarea {
+            @apply shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight text-xl;
```

Comment:
Let's remove the `shadow` around the form fields and keep them on the same plane as the form itself.